.devcontainer/Dockerfile.console

```diff
@@ -0,0 +1,17 @@
+FROM quay.io/openshift/origin-console:latest
```

Comment:
I'm debating if this should be `4.10` or `latest`. I'd guess most plugin developers would want to target a stable version (outside of Red Hat at least). Unfortunately, we don't have a `stable` tag.